Core/GDCore/Extensions/Builtin/SpriteExtension/SpriteExtension.cpp

```diff
@@ -211,6 +223,32 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsSpriteExtension(
       .UseStandardOperatorParameters("number")
       .MarkAsAdvanced();
 
+  obj.AddCondition("Height",
+                   _("Height"),
+                   _("Compare the height of a Sprite object."),
+                   _("the height"),
+                   _("Size"),
+                   "res/conditions/scaleHeight24.png",
+                   "res/conditions/scaleHeight.png")
+
+      .AddParameter("object", _("Object"), "Sprite")
+      .UseStandardRelationalOperatorParameters("number")
+      .MarkAsAdvanced();
+
+  obj.AddAction("ChangeSize",
+                _("Size"),
+                _("Change the size of an object."),
+                _("Change the size of _PARAM0_: _PARAM1_ (width) "
+                  ", _PARAM2_ (height)"),
```

Comment:
Parameter names usually follow this `name1: _PARAM1_, name2: _PARAM2_`, but for dimension a lighter syntax can be used.

The action sentences are not very consistent about this, I don't know which one to suggest:
* "Modify the size of a camera"
  "Change the size of camera _PARAM2_ of _PARAM1_ to _PARAM3_*_PARAM4_"

* "Change the size of the game window"
  "Set game window size to _PARAM1_x_PARAM2_ (also update game resolution: _PARAM3_)"

Maybe the cross character '✕' would be better?

Reply:
I would go for `Change the size of _PARAM0_: set to _PARAM1_x_PARAM2_` because:
- it mimics the traditional "Change [subject] of [object]: [operator] [value]". (you can consider the size as being an attribute, and its value is a dimension, so a tuple, and you can consider the operator is forced to be "=").